core/kazoo_voicemail/src/kz_vm_message.erl

```diff
@@ -43,6 +43,7 @@
 -define(MODB_COUNT_VIEW, <<"mailbox_messages/count_per_folder">>).
 -define(COUNT_BY_VMBOX, <<"mailbox_messages/count_by_vmbox">>).
 -define(BOX_MESSAGES_CB_LIST, <<"vmboxes/crossbar_listing">>).
+-define(PVT_LEGACY_TYPE, <<"private_media">>).
```

Comment:
macro is called `PVT_LEGACY_TYPE` but content reads `private_media`. I was wondering then why call it PVT_ ?